workflows_core/api/api.py

```diff
@@ -494,3 +492,81 @@ def _merge_tags(
             ),
         )
         return get_response(response)
+
+    @retry()
+    def _bulk_update_keyphrase(
+        self, dataset_id: str, field: str, alias: str, updates: List
+    ):
+        """
+        Update keyphrases
+        """
+        response = requests.post(
+            url=self._base_url
+            + f"datasets/{dataset_id}/fields/{field}.{alias}/keyphrase/bulk_update",
+            headers=self._headers,
+            json=dict(updates=updates),
+        )
+        return get_response(response)
+
+    @retry()
+    def _get_keyphrase(
+        self, dataset_id: str, field: str, alias: str, keyphrase_id: str
+    ):
+        """
+        Get keyphrase
+        """
+        response = requests.get(
+            url=self._base_url
+            + f"/datasets/{dataset_id}/fields/{field}.{alias}/{keyphrase_id}/get",
```

Comment:
should it be `"datasets/{dataset_id}/fields/{field}.{alias}/keyphrase/{keyphrase_id}/get"`?

Reply:
Same as above - thanks.